components/prize_pool/wikis/arenaofvalor/prize_pool_legacy_custom.lua

```diff
@@ -0,0 +1,43 @@
+---
+-- @Liquipedia
+-- wiki=arenaofvalor
+-- page=Module:PrizePool/Legacy/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local PrizePoolLegacy = Lua.import('Module:PrizePool/Legacy', {requireDevIfEnabled = true})
+
+local CustomLegacyPrizePool = {}
+
+-- Template entry point
+function CustomLegacyPrizePool.run()
+	return PrizePoolLegacy.run(CustomLegacyPrizePool)
+end
+
+function CustomLegacyPrizePool.customOpponent(opponentData, CACHED_DATA, slot, opponentIndex)
+	-- AoV has use case of same placement but has different earnings
+
+	if slot['usdprize' .. opponentIndex] then
+		opponentData.usdprize = slot['usdprize' .. opponentIndex]
+	end
+
+	if slot['localprize' .. opponentIndex] then
+		local param = CACHED_DATA.inputToId['localprize']
+		CustomLegacyPrizePool._setOpponentReward(opponentData, param, slot['localprize' .. opponentIndex])
+	end
+
+	return opponentData
+end
```

Comment:
Looks like only 3 pages had `localprizeX` or `usdprizeX`. Can we covert those three pages to new input format, and we can remove the entire legacy custom for AoV?

https://liquipedia.net/arenaofvalor/index.php?search=insource%3A%22localprize2%22&title=Special%3ASearch&profile=all&fulltext=1